scripting/pugsetup_teamnames.sp

```diff
@@ -135,6 +140,11 @@ static bool GetPlayerFlagFromIP(int client, char flag[3]) {
 public void FillPotentialNames(int team, ArrayList names, ArrayList flags) {
   for (int i = 1; i <= MaxClients; i++) {
     if (IsPlayer(i) && GetClientTeam(i) == team && AreClientCookiesCached(i)) {
+      if (g_UseCaptainNamesCvar.IntValue != 0 && PugSetup_GetTeamType() == TeamType_Captains) {
```

Comment:
`g_UseCaptainNamesCvar.BoolValue` would be fine here too

Reply:
Yup, although I don't think BoolValue is used at all in pugsetup so I won't require it to merge this :)